core/controllers/outgoing_emails.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for sending outgoing emails."""
+
+from core.controllers import base
+from core.domains import email_manager
+import json
+
+
+def write_email_contents(user_organization, user_comment):
```

Comment:
Perhaps declare these as @staticmethod in the handler if nothing else uses them.

Reply:
Done!